test/class_test.ts

```diff
@@ -1952,4 +1952,20 @@ describe('classes', () => {
       })();
     `);
   });
+
+  it('produce a proper hack to trick Babel and into TypeScript that would be OK with UglifyJS -- and continues to produce results as would be expected', () => {
```

Comment:
This test name is a bit long. There isn't a strict line length limit, but ~100 chars is typically a good target. Also ideally the test name completes a sentence starting with "it".

Maybe just something like "produces a Babel/TS constructor hack that allows this before super in a constructor". Really this is a general test for correctness for the hack stuff.

As for the uglify thing, I actually maybe wouldn't mention it here since this test doesn't actually check the uglify output. But it sounds like a good thing to mention in the comment at the top of babelConstructorWorkaroundLines.ts.

Reply:
Agreed, and the fact that minifiers treat it properly is just an implementation detail of this.